src/BookmarksManager.Chrome/BookmarksManager.Chrome.csproj

```diff
@@ -0,0 +1,16 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <Configurations>Debug;Release</Configurations>
+    <LangVersion>latest</LangVersion>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Newtonsoft.Json" Version="12.0.1" />
```

Comment:
Let's include the lowest .net standard Newtonsoft.Json version (e.g. 9.0.1).  This lib needs just a trivial JSON parsing functionality, so no point in requiring 12.0.1. This way nuget would not impose requirement for consumers already having 9.0.1 and up. 
I have already tested with 9.0.1 - works on my machine :)